edge-modules/MetricsCollector/config/settings.json

```diff
@@ -2,7 +2,7 @@
   "AzMonWorkspaceId": "",
   "AzMonWorkspaceKey": "",
   "AzMonLogType": "edgeHubMetrics",
-  "MetricsEndpoints": "[\"http://edgeHub:9600/metrics\"]",
+  "MetricsEndpoints": "http://edgeHub:9600/metrics",
```

Comment:
Did you check if this needs to be JSON escaped?

Reply:
It is a CSV. There are changes not showing as I need to sync branches.